Dockerfile

```diff
@@ -19,17 +19,15 @@ RUN apk add --update --no-cache \
         --enable-libx264 --enable-libx265 --enable-libopus --enable-libass \
         --enable-libwebp --enable-librtmp --enable-libtheora \
         --enable-libvorbis --enable-libvpx --enable-libxvid \
-    && make -j"$(nproc)" install \
-    && cd .. \
-    && rm -rf ffmpeg-${FFMPEG_VER} \
-    # cleanup
-    && apk del --purge \
-        coreutils build-base nasm curl tar openssl-dev zlib-dev yasm-dev \
-        lame-dev freetype-dev opus-dev xvidcore-dev libass-dev libwebp-dev \
-        libvorbis-dev libogg-dev libtheora-dev libvpx-dev \
-    && apk add --no-cache \
+    && make -j"$(nproc)" install
+
+# install ffmpeg and runtime dependencies
+FROM alpine:3.4
+
+COPY --from=build /usr/local/bin/ffmpeg /usr/local/bin/
+
+RUN apk add --no-cache \
         zlib lame freetype faac opus xvidcore libass libwebp libvorbis libogg \
-        libtheora libvpx \
-    && rm -rf /var/cache/apk/*
+        libtheora libvpx rtmpdump-dev x264-dev x265-dev ca-certificates
```

Comment:
`musl` should be added (it's already part of the image). Modify the `apk add --no-cache` back to `apk add --update --no-cache` and add `musl` to the list.